pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/sqlUtils.scala

```diff
@@ -0,0 +1,11 @@
+package edu.berkeley.cs.amplab.sparkr
```

Comment:
Nit: This file should be named SQLUtils.scala now (You can use `git mv <path>/sqlUtils.scala <path>/SQLUtils.scala`)